notification/src/config/config.js

```diff
@@ -1,6 +1,7 @@
-const { isEmpty } = require('lodash')
-const loadConfig = require('./config-loader')
+import lodash from 'lodash'
```

Comment:
```suggestion
import { isEmpty } from 'lodash'
```

Reply:
```
SyntaxError: Named export 'isEmpty' not found. The requested module 'lodash' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'lodash';
const { isEmpty } = pkg;
```
Had to do this way because of above error.